src/effects/recover.c

```diff
@@ -1,6 +1,14 @@
 #include "common.h"
 #include "effects_internal.h"
 
+void func_E0080448(EffectInstance* effect);
+
+s32 D_E0080AC0[] = { 0x09003540, 0x09003580, 0x090035C0, 0x09003600, 0x09003640, 0x09003680, 0x090036C0, 0x09003700, 0x09003740, 0x09003780, 0x090037C0, 0x09003800 };
+
+s32 D_E0080AF0[] = { 0x3E800000, 0x3F000000, 0x3F400000, 0x3F800000, 0x3F666666, 0x3F8CCCCD, 0x3F800000 };
```

Comment:
These are floats, so it's an f32 array just like the below one...weird that splat missed these. https://www.h-schmidt.net/FloatConverter/IEEE754.html